fgd/point/momentum/momentum_df_weaponspawner.fgd

```diff
@@ -0,0 +1,16 @@
+@PointClass base(BaseEntityPoint)
+	appliesto(MOMENTUM) 
+= momentum_df_weaponspawner : "Defrag weapon spawner."
+	[
+	weapon_slot[engine](integer) : "Weapon" : 2
+	weapon_slot(choices) : "Weapon" : 2 : "The weapon to spawn." =
+		[
+		2: "Machine gun"
+		3: "Knife"
+		4: "Grenade launcher"
+		5: "Rocket launcher"
+		8: "Plasma gun"
+		9: "BFG"
```

Comment:
Should be fine to go ahead and add "6: Lightning Gun" and "7: Railgun".

Right now 2 and 3 are accurate, but its rare that anyone adds weapon spawners for either of those weapons. In the future they will become "1: Knife" (or "Gauntlet" or whatever melee we have), "2: Machine Gun" and "3: Shotgun".

It's up to you if you want to add any of these now or leave it as-is.

Reply:
In my opinion it’s best to make those changes when they arrive, to reduce confusion until then